lib/application.js

```diff
@@ -284,7 +285,10 @@ app.handler = function(type, options) {
   var handler = this._handlers[type] = remotes.handler(type, options);
 
   remotes.classes().forEach(function(sharedClass) {
-    sharedClass.ctor.emit('mounted', app, sharedClass, remotes);
+    /* HACK!!! */
+    if (sharedClass.name !== 'ctrl') {
```

Comment:
I need the controller-ctor to inherit the EventEmitter functions and was trying to figure out a clean way of doing so.

I've seen this done before as follows:
```node
function ctor() {
    EventEmitter.call(this);
}

util.inherits(ctor, EventEmitter);
```

but in this case, `ctor` is the constructor created by the loopback-developer. I do not want the lb-developer to have to write this line, `EventEmitter.call(this);` inside their controller constructor.

Is there another way to do it?